src/main/java/org/jenkinsci/main/modules/instance_identity/InstanceIdentity.java

```diff
@@ -127,6 +130,15 @@ public RSAPrivateKey getPrivate() {
         return (RSAPrivateKey) keys.getPrivate();
     }
 
+    /**
+     * @since TODO
+     * @return the encoded RSA public key.
+     */
+    public String getEncodedPublicKey() {
+        RSAPublicKey key = getPublic();
+        return new String(Base64.encodeBase64(key.getEncoded()), StandardCharsets.UTF_8);
```

Comment:
```suggestion
        return new String(java.util.Base64.getEncoder().encode(key.getEncoded()), StandardCharsets.UTF_8);
```

(In addition to UTF_8 in the effort to avoid using third party library when code is available in JDK). Sorry for the full package name, I don't know if it's possible to manage such case easily with the new suggestion feature.

Reply:
I prefer to not mix change scopes in the same PR, I do not feel strongly about that one in general, but I do feel that this PR is not the place to do it.